dohq_artifactory/admin.py

```diff
@@ -1029,11 +1029,20 @@ class PermissionTarget(AdminObject):
     ROLE_ANNOTATE = (ANNOTATE, READ)
     ROLE_READ = READ
 
-    def __init__(self, artifactory, name, repositories=None, users=None, groups=None):
+    def __init__(
+        self,
+        artifactory,
+        name,
+        repositories=None,
+        users=None,
+        groups=None,
+        includesPattern="**",
+        excludesPattern="",
+    ):
```

Comment:
Looks good and backwards compatible, but let's add them as forced named arguments. This will help to maintain it in the future

Another thing, let's keep pep8 style. I understand why it is used mixed case here, but we have to change

```suggestion
    def __init__(
        self,
        artifactory,
        name,
        repositories=None,
        users=None,
        groups=None,
        *,
        includes_pattern="**",
        excludes_pattern="",
    ):
```